src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (num) => {
+    if (num) {
```

Comment:
will not work correct if num = 0

Reply:
yep, i checked and you were right)
i`ll fix this